client/ayon_core/tools/publisher/models/publish.py

```diff
@@ -869,6 +878,9 @@ def reset(self):
         # - pop the key after first collector using it would be safest option?
         self._publish_context.data["create_context"] = create_context
         publish_plugins = create_context.publish_plugins
+        for plugin in publish_plugins:
+            plugin.log.propagate = False
+
```

Comment:
Shouldn't we also do this in the iterator - because this would persist outside of the UI and would make the logs also NOT show outside of the publisher if a headless publish would be initiated after?

Reply:
Each `discovery` will reset all plugins.